### PR TITLE
Schedule: keep week 0 expanded during week 1 #243

### LINKED ISSUE #243: Schedule: keep week 0 expanded during week 1
Students can miss week 0 content if only week 1 is expanded in the first week. May be we treat week 0 as a special case and expand it during week 1 too?

### PR DESCRIPTION
**Pull request by acjh**
Fixes #243 
[Preview](http://rawgit.com/acjh/website/243-keep-week-0-expanded/)

### CODE REVIEW

File: scripts/common.js [hunk 1]
```diff
@@ -240,6 +240,12 @@ function loadContent(week) {
                 totalWeeks = 1;
             } else if (isCurrentWeek(week)) {
                 expandWeekFully(week);
+
+                if (week == 1) {
```
Comment (damithc) [inline scripts/common.js]: May be add a comment to explain this spacial case?

**Comment by acjh** [comment]
@damithc Shall I merge?

**Comment by damithc** [comment]
> Shall I merge?

Yes, go ahead 👍

**Comment by damithc** [comment]
@hwkchia FYI

**Pull request merged**